docs/manual/developer_guide.adoc

```diff
@@ -555,8 +555,50 @@ Contributions can be made for rules, checks, remediations or even utilities. The
+
+Rules describe the desired state of the system and may contain references if they are parts of higher-level standards.
+For example, the `partition_for_tmp` rule from the mentioned file is part of STIG, so definition of the rule contains the `ref` reference to it.
+
+A rule itself has these attributes:
+
+* `id`: The primary key for the rule. It is referenced by profiles.
+* `severity`: Can be `low`, `medium`, or `high`.
+
+A rule contains those elements that are text-centric:
+
+* `title`: Human-readable title of the rule.
+* `rationale`: Human-readable HTML description of the reason why the rule exists and why it is important from the technical point of view. For example, rationale of the `partition_for_tmp` rule states that:
++
+The <tt>/tmp</tt> partition is used as temporary storage by many programs. Placing <tt>/tmp</tt> in its own partition enables the setting of more restrictive mount options, which can help protect programs which use it.
+* `description`: Human-readable HTML description, which provides broader context for non-experts than the rationale. For example, rationale of the `partition_for_tmp` rule states that:
++
+The <tt>/tmp</tt> partition is used as temporary storage by many programs. Placing <tt>/tmp</tt> in its own partition enables the setting of more restrictive mount options, which can help protect programs which use it.
+* `ocil`: Defines assert statements. The `clause` attribute contains the statement, and the element text describes how to determine whether the statement is true or false. Check out rule `encrypt_partitions` in `shared/xccdf/system/software/disk_partitioning.xml`, that contains the `partitions do not have a type of crypto_LUKS` clause.
+
+A rule may contain those reference-type elements:
+
+* `oval`: Link to the check that is able to determine whether the scanned system complies to the rule. Checks are written in the OVAL language (see the section Checks for further info).
+* `ident`: This is related to products/CCEs that the rule applies to.
++
+When the rule is related to RHEL, it should have a CCE.
+Available CCEs that can be assigned to new rules are listed in the `shared/references/cce-rhel-avail.txt` file.
+* `ref`: If the rule is part of a standard, it is referenced using the `ref` element. One rule can have multiple `ref` elements.
+
+Some of existing rule definitions contain elements that use macros.
+You can find definitions of those macros in one of the files in the `shared/transforms` directory.
+For example, the `ocil` element of `partition_for_tmp` uses the `partition-check-macro`, which is defined in `shared/transforms/shared_shorthand2xccdf.xslt`.
+
+Rules are unselected by default - even if the scanner knows about them, they are effectively ignored during the scan or remediation.
+A rule may be selected by a profile, so when the scanner is scanning using a profile the rule belongs to, the rule is taken into account.
```

Comment:
I'm not a fan of `belongs to`, it gives the impression that a Rule can belong to only one Profile. Maybe `integrates` or `is part of`.
And/Or mention that a Rule can be selected by multiple Profiles.

> the rule is taken into account

the rule is evaluated?

Reply:
Thank you for you suggestions, I identify with them. There is just one exception - I prefer the "taking into account" phrase, because rule is just a definition, there is nothing to evaluate. Checks and remediations can be evaluated, but I feel that it is different from rules.

Reply:
Okay.